forms.rst

```diff
@@ -676,7 +676,19 @@ choice is ultimately up to you.
 .. note::
 
     The form name is automatically generated from the type class name. If you want
-    to modify it, use the :method:`Symfony\\Component\\Form\\FormFactoryInterface::createNamed` method.
+    to modify it, use the :method:`Symfony\\Component\\Form\\FormFactoryInterface::createNamed` method::
+
+        // src/AppBundle/Controller/DefaultController.php
+        use AppBundle\Form\TaskType;
+
+        public function newAction()
+        {
+            $task = ...;
+            $form = $this->container->get('form.factory')->createNamed('name', TaskType::class, $task);
```

Comment:
For short, we can use the shortcut instead `$this->get(...)`.

Reply:
Yes that’s true, do we have a “standard” in the docs?

Reply:
The recommended way is to inject the factory with dependency injection but that will make this example maybe to complex.
```$this->container``` looks in this case more clearly than ```$this->get```

Reply:
I also think using `$this->get(...)` makes sense here.

@wimme002 as long as you extend `AbstractController` this service is available in the service locator:

https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php#L86

So imho no need to inject it.

Reply:
Agreed!

Reply:
Made the requested changes